src/Framework/Exceptions/UncaughtExceptionLogger.php

```diff
@@ -53,15 +53,6 @@ public function handleException( Exception $exception ) {
 	 * @return bool
 	 */
 	private function isGiveException( Exception $exception ) {
-		if ( $exception instanceof LoggableException ) {
-			return true;
-		}
-
-		$traits = class_uses( $exception );
-		if ( isset( $traits[ Loggable::class ] ) ) {
-			return true;
-		}
-
-		return false;
+		return $exception instanceof LoggableException;
```

Comment:
As this is simplified, it may not be necessary to have a separate method. "If the exception is an instance of LoggableException" is self-explanatory - or at least as readable as `isGiveException` (which isn't necessarily accurate at this point, rather the point is that the exception is loggable - and now I'm rambling...).

Reply:
Cleaned up in b411819